package.json

```diff
@@ -26,10 +26,9 @@
   },
   "homepage": "https://github.com/agco/passport-openam",
   "dependencies": {
-    "MD5": "^1.2.1",
-    "bluebird": "^2.9.24",
+    "bluebird": "^3.3.4",
     "http-as-promised": "^1.0.0",
-    "lodash": "^3.7.0",
+    "md5": "^2.1.0",
```

Comment:
these are significant changes. Are you aware of what the changes are, and what the impact is?